src/hub.js

```diff
@@ -273,16 +273,22 @@ const onReady = async () => {
           mediaStream.removeTrack(track);
         }
       }
-      NAF.connection.adapter.setLocalMediaStream(mediaStream);
+      if (NAF.connection.adapter && NAF.connection.adapter.setLocalMediaStream) {
```

Comment:
can this be simplified to check only `NAF.connection.adapter`?

Reply:
I think we wanted to check for `setLocalMediaStream` specifically, in case we were ever using anything other than `naf-janus-adapter`. `setLocalMediaStream` is special to our adapter.

Reply:
okay, yeah, that's along what I thought